### DISCUSSION TITLE
Nodes not emitting value on startup

### DISCUSSION THREAD

**Discussion by jbscherrer** [bonsai-rx/bonsai#1100] [Q&A]
Can anyone tell me why the second set of add/subtract nodes are not producing elements on startup? Is there something really obvious I'm missing?

Screenshot is from `Motion_Correct_ROI` in the main workflow of `ncaf.bonsai` in the attached zip file.

Thanks!
![image](https://user-images.githubusercontent.com/39060975/198119017-4bd3fee6-54a0-4b84-b2e1-f3130acfddf6.png)

[bonsai_CAF.zip](https://github.com/bonsai-rx/bonsai/files/9873135/bonsai_CAF.zip)

**Comment by bruno-f-cruz** [comment]
I can't seem to open your file. I am not sure what version of the NiDaq package you have installed but it crashes with mine. Is there any chance you can send a barebones examples without references to the nidaq package that still reproduces your problem? Either that or send me your Bonsai.config file so I can try to figure out what package version you are using. 
Cheers,
Bruno

**Reply by jbscherrer** [reply]
Here's my bonsai.config file. It looks like I'm using version 0.1.0, which is the most recent release published by Gonçalo.

[Bonsai.zip](https://github.com/bonsai-rx/bonsai/files/9911271/Bonsai.zip)
